src/js/util.js

```diff
@@ -673,19 +673,58 @@
             return false;
         },
 
-        cleanListDOM: function (ownerDocument, element) {
+        cleanListDOM: function (contentWindow, ownerDocument, element) {
```

Comment:
This is a breaking change as you are changing the method signature. Could you better put that new parameter at the end and handle the case where that parameter won't be given?